runtime/v2/app.go

```diff
@@ -123,3 +124,11 @@ func (a *App[T]) GetAppManager() *appmanager.AppManager[T] {
 func (a *App[T]) GetGPRCMethodsToMessageMap() map[string]func() gogoproto.Message {
 	return a.GRPCMethodsToMessageMap
 }
+
+func (a *App[T]) Query(ctx context.Context, gasLimit, version uint64, req transaction.Msg) (transaction.Msg, error) {
```

Comment:
i couldn't find other to get bonded validator directly because of `executionContext`. Dont know if i missed something here

Reply:
We can use the staking keeper directly and remove this, like in export v1.

Reply:
can't we pass the correct context from the calling func?